### ISSUE TITLE
Naming

### ISSUE THREAD

**Issue by ab-pm** [Wideplink/eslint-plugin-named-import-spacing#42]
I'd love to use this rule, as it allows me to enforce the spacing in import declarations without affecting arrays and object literals (as [object-curly-spaces](https://eslint.org/docs/latest/rules/object-curly-spacing) does, not having a separate option for imports).

However, the syntax of imports has nothing to do with destructuring, rather it's called [*NamedImports*](https://262.ecma-international.org/13.0/#prod-NamedImports), making the whole statement a named import declaration. So `destructuring-import-spacing` doesn't really fit as a name :-(

**Comment by mtripg6666tdr** [comment]
Hi, first of all you're right. 
The name came from just its look of syntax, which is similar with ES6 destructuring.
> If you want to run on Node.js the file that contains import statements of ES Module, you generally have to compile with some tool like babel or tsc and will get CommonJS files that uses actual destructuring syntax.
Anyway, It's sure that I mixed up names of these two syntax when I created this plugin, I'm sorry for the confusing name.
I'm going to consider whether or not releasing with the better name in future, when I'm not so busy!
Thank you for opening the issue and notified me!

**Comment by mtripg6666tdr** [comment]
Hi, first of all you're right. 
The name comes from just its look of syntax, which is similar with ES6 destructuring.
> If you want to run on Node.js the file that contains import statements of ES Module, you generally have to compile with some tool like babel or tsc and will get CommonJS files that uses actual destructuring syntax.
Anyway, It's sure that I mixed up names of these two syntax when I created this plugin, I'm sorry for the confusing name.
I'm going to consider whether or not releasing with the better name in future, when I'm not so busy!
Thank you for opening the issue and notified me!
